clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -7706,7 +7707,7 @@ void OffloadBundler::ConstructJob(Compilation &C, const JobAction &JA,
     }
     UB += CurTC->getInputFilename(Inputs[I]);
   }
-  // For -fintelfpga, when bundling objects we also want to bundle up the
+  // For spir64_fpga, when bundling objects we also want to bundle up the
```

Comment:
Not sure if that is consistent with the intention of the change (dedicate `-fintelfpga` solely to dependency handling)

Reply:
Yup, good catch.  I'll change it back.